src/anbox/utils.cpp

```diff
@@ -193,5 +194,17 @@ bool is_mounted(const std::string &path) {
 
 }
 
+std::string find_program_on_path(const std::string &name) {
```

Comment:
Naming is a little bit confusing, something like: get_full_path (or just "which" :) imho would be more descriptive about what is actually being returned.